rosco-web/rosco-web.gradle

```diff
@@ -1,4 +1,5 @@
 apply plugin: 'io.spinnaker.package'
+apply plugin: 'io.spinnaker.artifactregistry-publish'
```

Comment:
I'm not 100% sure what the full process for release a new version of the gradle plugins is but that would have to happen and then bump the version in rosco also to find this new plugin

Reply:
This is all taken care of...